packages/react-utilities/src/hooks/useOverrideNativeKeyboardClick.ts

```diff
@@ -0,0 +1,94 @@
+import * as React from 'react';
+import { getCode, SpacebarKey, EnterKey } from '@fluentui/keyboard-key';
```

Comment:
@smhigley @kolaps33  Any feedback from you about native click/keyboard handling ?